metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -612,28 +612,21 @@ protected UpdateAspectResult patchAspectToLocalDB(
     return _aspectDao.runInTransactionWithRetry(() -> {
       final String urnStr = urn.toString();
       final String aspectName = aspectSpec.getName();
-      EntityAspect latest = _aspectDao.getLatestAspect(urnStr, aspectName);
-      if (latest == null) {
-        //TODO: best effort mint
-        RecordTemplate defaultTemplate = _entityRegistry.getAspectTemplateEngine().getDefaultTemplate(aspectSpec.getName());
-
-        if (defaultTemplate != null) {
-          latest = new EntityAspect();
-          latest.setAspect(aspectName);
-          latest.setMetadata(EntityUtils.toJsonAspect(defaultTemplate));
-          latest.setUrn(urnStr);
-          latest.setVersion(ASPECT_LATEST_VERSION);
-          latest.setCreatedOn(new Timestamp(auditStamp.getTime()));
-          latest.setCreatedBy(auditStamp.getActor().toString());
-        } else {
+      final EntityAspect latest = _aspectDao.getLatestAspect(urnStr, aspectName);
```

Comment:
Confused on this change, creating the default template is non-trivial so it's within this block to avoid unnecessarily creating it. There is also missing information when filling it out just using the default template that needs to be added to prevent issues.

The core of what is happening here is ensuring that required fields are filled out. Patching on non-existing rows works for the currently set up patches so I'm a little confused on why it wouldn't for the new aspect.

Reply:
It does not for any aspects